app/filters.js

```diff
@@ -179,6 +179,17 @@ module.exports = (env) => {
     else return numbers[number -1]
   }
 
+  filters.ordinal = (number) => {
+    const ordinals = [ 'first', 'second', 'third', 'fourth', 'fifth', 'sixth', 'seventh', 'eighth', 'ninth' ];
```

Comment:
What happens for tenth?

Reply:
We spell out first to ninth. After that, we use 10th, 11th and so on.

In tables, we should use numerals throughout.

See GOV.UK style guide: https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style#ordinal-numbers

Reply:
Comment added to the filter code